homeassistant/components/lcn/switch.py

```diff
@@ -39,46 +50,48 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class LcnOutputSwitch(LcnEntity, SwitchEntity):
     """Representation of a LCN switch for output ports."""
 
-    def __init__(self, config, entry_id, device_connection):
+    def __init__(
+        self, config: ConfigType, entry_id: str, device_connection: DeviceConnectionType
+    ) -> None:
         """Initialize the LCN switch."""
         super().__init__(config, entry_id, device_connection)
 
         self.output = pypck.lcn_defs.OutputPort[config[CONF_DOMAIN_DATA][CONF_OUTPUT]]
 
-        self._is_on = None
+        self._is_on = False
```

Comment:
Shouldn't it be `None` until the state is known?
You can hint that self._is_on is Optional(bool)
```suggestion
        self._is_on :bool|None = None
```

Reply:
By definition the `is_on` property of `ToggleEntity` has return type `bool`. So I gues the initial value should also be of that type and not `None`.